libvirt.go

```diff
@@ -94,6 +95,75 @@ type qemuError struct {
 // DomainXMLFlags specifies options for dumping a domain's XML.
 type DomainXMLFlags uint32
 
+// Consts used for flags
+
+// virDomainModificationImpact and virTypedParameterFlags
+const (
+	FlagDomainAffectCurrent = 0
+	FlagDomainAffectLive    = 1 << (iota - 1)
+	FlagDomainAffectConfig
+	FlagTypedParamStringOkay
+)
+
+// Consts relating to TypedParams:
+const (
+	_ = iota
```

Comment:
this can go away, using `iota + 1` on the next line

Reply:
Cool, I like that better.